README.md

```diff
@@ -1,5 +1,46 @@
 # AutoML
 
-We're designing the AutoML platform that would make it easy for non-technical people to build custom AI solutions and AI-powered process automation.
+We're designing the AutoML platform that would make it easy for non-technical people to build custom AI solutions and
+AI-powered process automation.
 
-Easy-to-use ML platforms leverage the time/value/knowledge trade-off in a genuinely attractive way and allow users with no AI coding skills to optimize day-to-day operations and to solve business issues.
+Easy-to-use ML platforms leverage the time/value/knowledge trade-off in a genuinely attractive way and allow users with
+no AI coding skills to optimize day-to-day operations and to solve business issues.
+
+## API
+
+API Link: Todo
+
+## Quick Start
+
+### How kubernetes deploys AutoML
+
+需要开放以下端口：Todo
+
+#### Step 1: 部署依赖环境
+
+- Minikube
+- Docker
+
```

Comment:
By the way, could you please also add the steps about installing the prerequisites. Below is an example
https://github.com/ml-for-good/automl/blob/f0c6cd8fffbbdc962f043928cdc2a79596d98c99/Investigation/python/deploy/README.md